calyx-opt/src/passes/simplify_with_control.rs

```diff
@@ -268,7 +268,7 @@ impl Visitor for SimplifyWithControl {
     ) -> VisResult {
         if comp.is_static() {
             let msg =
-                format!("Component {} has both a top-level \"static\" annotations and combinational groups which is not supported", comp.name);
+                format!("Component {} has both a top-level <static> annotations and combinational groups which is not supported", comp.name);
```

Comment:
`<static>` is not an annotation. Let's rewrite this message to say "Static component {} ..."